### DISCUSSION TITLE
Naiver Stokes Module IntegralDirectedSurfaceForce

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#26181] [Q&A Modules: General] [answered]
I am getting the following error: A 'IntegralDirectedSurfaceForce' is not a registered object. There seems to be a Naiver Stokes Module associated with the postprocessor. What do I need to do to get access to this post processor? I am performing the modeling in the Combined Module.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Combined module should include the NavierStokes module. So it should be available.
Which executable are you using? and can you check with ldd (otool on mac) to see if NS is properly linked in

Guillaume

**Reply by TLWise** [reply]
I looked over the code for IntegralDirectedSurfaceForce and I see that it was developed only for use with the finite volume code of NS.

**Reply by GiudGiud** [reply]
Well even if it's designed for Finite volume, you should not get a "not registered object" error. 
But if you are not going to use it then I guess it doesnt matter anymore
